levels.go

```diff
@@ -1512,12 +1512,11 @@ func (s *levelsController) doCompact(id int, p compactionPriority) error {
 		p.t = s.levelTargets()
 	}
 
-	_, span := otrace.StartSpan(context.Background(), "Badger.Compaction")
+	_, span := otel.Tracer("Badger").Start(context.TODO(), "Compaction")
```

Comment:
For backward compatibility, let's use the same name as before:
```
otel.Tracer("").Start(context.Background(), "Badger.Compaction")
```

Reply:
Changed to the proposed names, thanks!